pkg/volume/util/device_util_linux.go

```diff
@@ -80,3 +83,202 @@ func (handler *deviceHandler) FindSlaveDevicesOnMultipath(dm string) []string {
 	}
 	return devices
 }
+
+// GetIscsiPortalHostMapForTarget given a target iqn, find all the scsi hosts logged into
+// that target. Returns a map by target portal to
```

Comment:
Can you please expand the source code comment on how the portalmap looks like ?

Reply:
Fixed